packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -367,7 +367,7 @@ curl -v -X POST \
       "buttonSelector": "#btn-login",
       "passwordSelector": "#txtbox-password",
       "userNameSelector": "#txtbox-username",
-      "targetUrl": "https://example.com/login.html",
+      "targetURL": "https://example.com/login.html",
```

Comment:
Annoying that this is inconsistent with `loginUrlRegex` below...

Reply:
agreed :(